docs/api.yaml

```diff
@@ -0,0 +1,1061 @@
+openapi: "3.0.3"
+info:
+  #  description:
```

Comment:
remove

Reply:
Do you mean the comments ?

Reply:
yeah, never put "draft" comments in code